--- ext_tables_static+adt-orig.sql	2007-02-08 09:03:38.000000000 +0100
+++ ext_tables_static+adt.sql	2007-02-28 15:38:35.000000000 +0100
@@ -1,5 +1,6 @@
 # TYPO3 Extension Manager dump 1.1
 #
+# Host: localhost    Database: fholzinger
 #--------------------------------------------------------
 
 
@@ -47,9 +48,9 @@
 INSERT INTO static_territories VALUES ('30', '0', '34', '142', 'Southern Asia');
 
 
-
 # TYPO3 Extension Manager dump 1.1
 #
+# Host: localhost    Database: fholzinger
 #--------------------------------------------------------
 
 
@@ -322,11 +323,14 @@
 INSERT INTO static_countries VALUES ('239', '0', 'CS', 'CSG', '891', '39', 'Државна заједница Србија и Црна Гора', 'State Union of Serbia and Montenegro', 'Belgrade', 'cs', 'CSD', '891', '381', '0', '0', '0', 'Србија и Црна Гора', 'Serbia and Montenegro', '1');
 INSERT INTO static_countries VALUES ('240', '0', 'AX', 'ALA', '248', '154', 'Åland Islands', 'Åland Islands', 'Mariehamn', 'fi', 'EUR', '978', '35818', '1', '0', '0', 'Åland Islands', 'Åland Islands', '0');
 INSERT INTO static_countries VALUES ('241', '0', 'HM', 'HMD', '334', '53', 'Heard Island and McDonald Islands', 'Heard Island and McDonald Islands', '', '', 'AUD', '36', '0', '0', '0', '0', 'Heard Island and McDonald Islands', 'Heard Island and McDonald Islands', '0');
-
+INSERT INTO static_countries VALUES ('242', '0', 'ME', 'MNE', '499', '39', 'Republika Crna Gora', 'Republic of Montenegro', 'Podgorica', 'me', 'EUR', '978', '382', '0', '0', '0', 'Crna Gora', 'Montenegro', '1');
+INSERT INTO static_countries VALUES ('243', '0', 'RS', 'SRB', '688', '151', 'Republika Srbija', 'Republic of Serbia', 'Belgrade', 'rs', 'RSD', '941', '381', '0', '0', '0', 'Serbia', 'Serbia', '1');
+INSERT INTO static_countries VALUES ('244', '0', 'GG', 'GGY', '831', '155', 'Bailiwick of Guernsey', 'Bailiwick of Guernsey', 'St Peter Port', 'gg', 'GBP', '826', '44', '0', '0', '0', 'Guernsey', 'Guernsey', '0');
 
 
 # TYPO3 Extension Manager dump 1.1
 #
+# Host: localhost    Database: fholzinger
 #--------------------------------------------------------
 
 
@@ -819,9 +823,9 @@
 INSERT INTO static_country_zones VALUES ('482', '0', 'BR', 'BRA', '76', 'TO', 'Tocantins', '');
 
 
-
 # TYPO3 Extension Manager dump 1.1
 #
+# Host: localhost    Database: fholzinger
 #--------------------------------------------------------
 
 
@@ -1015,9 +1019,9 @@
 INSERT INTO static_currencies VALUES ('175', '0', 'TRY', '949', 'Turkish new lira', 'YTL', '', '.', ',', '2', 'new kuruş', '100', '', '');
 
 
-
 # TYPO3 Extension Manager dump 1.1
 #
+# Host: localhost    Database: fholzinger
 #--------------------------------------------------------
 
 
@@ -1231,9 +1235,9 @@
 INSERT INTO static_languages VALUES ('182', '0', 'PI', 'Pali', '', '', '', 'Pāli', '1', '0');
 
 
-
 # TYPO3 Extension Manager dump 1.1
 #
+# Host: localhost    Database: fholzinger
 #--------------------------------------------------------
 
 
@@ -1301,4 +1305,3 @@
 INSERT INTO static_taxes VALUES ('35', '0', '1078709361', '0', '0', '0', '0', '484', 'MX', 'MEX', '', 'Mexico VAT', '1', 'IVA', '3', '0.15', '1');
 
 
-
